version.js

```diff
@@ -16,6 +17,9 @@ const gitInfo = gitDescribeSync({
 // Append edition information
 gitInfo.edition = getEditionDisplayName();
 
+// Re-use the version logic from our Makefile
+gitInfo.ui = execSync("make version --no-print-directory").toString().trim();
```

Comment:
nit: I'd maybe change `ui` to something like `humanReadable`, `humanFriendly`, `readable`.

@kgroschoff could you check if this doesn't conflict with your workflow? As you are working on windows I am kind of worried it could break something for you.

Reply:
Does not break something, but the version isn't entirely correct for me ;) `"ui": "v2.14.0-257-g15c8a66b"`
I personally don't have a big problem with it, as it is only local 🤷

Reply:
FYI, the "wrong" version will also be generatedon linux if you're not on the master branch. The Makefile logic is setup to work primarily for CI (i.e. where it matters). To make it work in local release branches, more logic would be necessary and I think none of us want to make the Makefile even more complicated ^^